pandas/tests/frame/test_indexing.py

```diff
@@ -208,6 +208,43 @@ def test_setitem_list_of_tuples(self, float_frame):
         expected = Series(tuples, index=float_frame.index, name="tuples")
         assert_series_equal(result, expected)
 
+    def test_setitem_list_all_missing_columns_scalar(self, float_frame):
```

Comment:
can you parameterize these, typicall we add an arg called box to do this, eg.

```
@pytest.mark.parameterize('box', [lambda x: 1, lambda x: [1, 2], lambda x: x[['B', 'C']]])
...
```

Reply:
I can parametrize the right-hand size of an assignment but how do I parametrize the left-hand size?